deploy/vagrant/provision/roles/ufs_hadoop2/files/compile_tachyon.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 cd /tachyon
-mvn clean install -Dmaven.javadoc.skip=true -DskipTests -Dhadoop.version=${HADOOP_VERSION} -Dlicense.skip=true -Dcheckstyle.skip=true >mvn.log 2>&1
+mvn clean install -Dmaven.javadoc.skip=true -DskipTests -Dhadoop.version=${HADOOP_VERSION} -Dlicense.skip=true -Dcheckstyle.skip=true -Pmesos -Pyarn >mvn.log 2>&1
```

Comment:
YARN requires hadoop.version>=2.4.0, otherwise the build process will fail. Probably remove yarn?